TFT/src/User/API/Language/language_cz.h

```diff
@@ -335,7 +335,7 @@
     #define STRING_MPC_TITLE              "MPC autokalibrace"
     #define STRING_PID                    "PID"
     #define STRING_PID_TITLE              "PID autokalibrace"
-    #define STRING_TUNE_START_INFO        "Autokalibrace bude spuštěna. Bude to chvíli trvat.\nPokračovat?"
+    #define STRING_TUNE_START_INFO        "Spustit kalibraci vytápění? Bude to chvíli trvat.\nPokračovat?"
```

Comment:
Double question? To which one is the user responding? What if the user doesn't want to run heater calibration but he wants to continue? :)

Reply:
Yes you're right. They are two questions about the same thing, but it could be misunderstood. I will fix it.